feathr_project/feathr/version.py

```diff
@@ -7,4 +7,4 @@ def get_version():
 import os
 def get_maven_artifact_fullname():
     maven_artifact_version = os.environ.get("MAVEN_ARTIFACT_VERSION", __version__)
-    return f"com.linkedin.feathr:feathr_2.12:{maven_artifact_version}"
\ No newline at end of file
+    return f"com.linkedin.feathr:feathr_2.12:{maven_artifact_version}"
```

Comment:
Why are all these files from the `feathr_project` module deleted?

Reply:
Added them back